doc/mgmt/Docker to Host communication.md

```diff
@@ -2,7 +2,7 @@
 Docker to Host Communications
 
 # High Level Design Document
-#### Rev 0.7
+#### Rev 0.8
```

Comment:
How does DBus host service throw an exception to the client application?

Reply:
Dbus can't throw exception to client application, and we can use return value. For example, below api has int and string for return value.

    @host_service.method(host_service.bus_name(MOD_NAME), in_signature='s', out_signature='is')
    def apply_patch_db(self, patch_file_path):